test/react-native-cli/features/steps/steps.rb

```diff
@@ -2,18 +2,18 @@
 require 'securerandom'
 
 fixtures = Dir["#{__dir__}/../fixtures/rn0_*"].map { |dir| File.basename(dir) }.sort
-current_fixture = ENV['REACT_NATIVE_VERSION']
+$current_fixture = ENV['REACT_NATIVE_VERSION']
```

Comment:
The Ruby-n00bie in me struggled to access this variable inside of the new `get_android_main_application_path` function I wrote without making it global. I'm sure there's a nicer way to do it.

Reply:
`get_android_main_application_path` seems to do the same thing regardless of `current_fixture` — is it no longer useful?

I think the best & easiest way to make it not-global would be to pass it as a parameter

Reply:
It was doing the same thing due to a copy pasta error :) I fixed the logic in c3a630d. I'll unglobalise it and pass `current_fixture` as a parameter.